test/viz/validation.spec.ts

```diff
@@ -26,6 +26,26 @@ test('parseBucketName_bucketWithChildDirectories', () => {
   expect(actual).toEqual('gs://bucketName');
 });
 
+test('parseBucketName_extraSlashes', () => {
+  expect(() => sut.parseBucketName('gs://bucketName//')).toThrow(
+    'invalid gcs bucket name'
+  );
+});
+
+test('parseBucketName_bigPath', async () => {
+  expect(() =>
+    sut.parseBucketName(
+      'gs://bucketName/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa//aaaaaaaaaa/'
+    )
+  ).toThrow('invalid gcs bucket name');
+});
+
+test('parseBucketName_extraSlashesNested', () => {
```

Comment:
Nit: test names should be human readable sentences, not long identifiers. This is one of the things I think is unequivocally preferable in JS testing frameworks over conventional ones in other languages.